node-api/handlers/beacon/types/response.go

```diff
@@ -141,3 +142,28 @@ type Sidecar struct {
 type SidecarsResponse struct {
 	Data []*Sidecar `json:"data"`
 }
+
+// PendingPartialWithdrawalsResponse has a version field to indicate the fork version.
+// https://ethereum.github.io/beacon-APIs/#/Beacon/getPendingPartialWithdrawals
+type PendingPartialWithdrawalsResponse struct {
+	Version string `json:"version"`
+	GenericResponse
+}
+
+type PendingPartialWithdrawalData struct {
+	ValidatorIndex  uint64 `json:"validator_index,string"`
+	Amount          uint64 `json:"amount,string"`
+	WithdrawalEpoch string `json:"withdrawal_epoch"`
```

Comment:
Why only the first 2 have "string" json tag and the last one is type string? It seems in https://ethereum.github.io/beacon-APIs/#/Beacon/getPendingPartialWithdrawals that all three fields are decimal numbers returned as string. Just feel all 3 fields should be consistent

Reply:
fair point, addressed, keeping everything as uint64 and json tag as "string"